pelicanconf.py

```diff
@@ -66,6 +66,8 @@
 THEME = "theme/Flex"
 PLUGINS = ["i18n_subsites"]
 PYGMENTS_STYLE = "default"
+THEME_TEMPLATES_OVERRIDES = ["theme/Flex-extend/templates"]
+THEME_STATIC_PATHS = ["static", "../Flex-extend/static"]
```

Comment:
```suggestion
THEME_STATIC_PATHS = ["../Flex-extend/static"]
```

Just tested locally. we might not need `static` here